### PR TITLE
feat(icons): new icons march and april

### PR DESCRIPTION
**Pull request by conradennis**
Submitting new and updated UI icons from March and April 2022
all updates and additions approved by BXD

16 net new icons
1 updated icon
1 deprecated icon
updated categories.yml with additions
updated ui-icon-master.ai with additions and updates
updated deprecated.yml with removed icon (IDK if I did this right...)

#### Changelog

**Net New (16 total)**
– 32/calendar--add.svg
– 32/calendar--add--alt.svg
– 32/data-enrichment--add.svg
– 32/data-quality-definition.svg
– 32/ibm--open-enterprise-languages.svg
– 32/ibm-cloud-pak--business-automation.svg
– 32/ibm-cloud-pak--MANTA-automated-data-lineage.svg
– 32/ibm-z--cloud-mod-stack.svg
— 32/ibm-z-cloud--provisioning.svg
– 32/ibm-z-os--package-manager.svg
– 32/logo--kubernetes.svg
– 32/microservices--1.svg
– 32/microservices--2.svg
– 32/rule--data-quality.svg
– 32/service-id.svg
– 32/workflow-automation.svg

**Updated**
– 32/data-enrichment.svg

**Deprecated**
– 32/kubernetes.svg

`kubernetes.svg` is being deprecated in favor of the new `logo--kubernetes.svg` which complies with Kubernetes logo usage and brand standards

### CODE REVIEW

File: packages/icons/deprecated.yml [hunk 1]
```diff
@@ -1 +1,6 @@
 deprecated: []
+- name: Organization
+    subcategories:
+      - name: Systems
+        members:
+          - kubernetes
```
Comment (jnm2377) [inline packages/icons/deprecated.yml]: it looks like ci doesn't like something about the formatting here, can you fix the formatting?
Screen Shot 2022-05-09 at 3 28 02 PM

Comment (tay1orjones) [reply packages/icons/deprecated.yml]: Based on [what this file previously used to look like](https://github.com/conradennis/carbon/commit/eda5c7de6591b1ddc36a31c2577f35e57c6b2ca5#diff-14365a39de10c1ccfde6b7ee38b7fa6173870479719eae6877adcc80ed49c6f8), it appears this file should be just names and reasons for deprecations.


```yml
deprecated:
  - name: app-switcher
    reason: The app switcher icon has been replaced by switcher
  - name: arrows
    reason: The arrows icon has been replaced by arrows--vertical
  # ...etc
```

Comment (conradennis) [reply packages/icons/deprecated.yml:6]: @jnm2377 sure, any insight on how I should format it? When I open the deprecated.yml file it just says:

Deprecated:[]

so should I put the icon name in the brackets and I'm good to go? Sorry not familiar with this!

Comment (conradennis) [reply packages/icons/deprecated.yml]: @jnm2377 thanks for the heads up
@tay1orjones thanks for sharing, I'll reformat as shown above.

**Review by joshblack — APPROVED** [review]
Looks good after the `deprecated.yml` changes!

**Review by laurenmrice — APPROVED** [review]
Looks good! 🎉

**Comment by conradennis** [comment]
@laurenmrice @joshblack I made the changes, hopefully the update to the deprecated.yml file is correct. Let me know if I need to make any further fixes, thanks

**Comment by tay1orjones** [comment]
@conradennis the build is failing with an error:

```
@carbon/icons: Error: Expected the icon `calendar--add--alt` to be defined in the data metadata file. Found matches for this asset in the following locations:
@carbon/icons: /home/runner/work/carbon/carbon/packages/icons/src/svg/32/calendar--add--alt.svg
```

Does `packages/icons/icons.yml` need to be updated to include `calendar--add--alt` along with any of the other new ones?

**Comment by conradennis** [comment]
@tay1orjones yup, our process has been that once the PR is merged someone from the Blue Studio content team updates the `packages/icons/icons.yml` file with metadata for each new icon

**Comment by joshblack** [comment]
@tay1orjones I think the steps are listed out over in: https://github.com/carbon-design-system/carbon/blob/main/docs/developer-handbook.md#working-with-icons-and-pictograms

Basically on our end we need to run `node tasks/scaffold.js` in order to make sure the metadata file is up to date.

**Comment by conradennis** [comment]
I have a last minute request from a team, need to include a couple icons before merging. Will push the updates in a few minutes.

**Review by jnm2377 — APPROVED** [review]

**Pull request merged**